do.sh

```diff
@@ -338,11 +379,14 @@ function run_test() {
     # Perform a fast cleanup by doing a minimal redeploy.
     init_ovn_fake_multinode
 
-    source ${rundir}/${ovn_heater_venv}/bin/activate
-    pushd ${out_dir}
+    tester_ip=$(get_tester_ip ${test_file})
+    if ! ansible-playbook ${ovn_fmn_playbooks}/run-tester.yml -i ${hosts_file} --extra-vars "test_file=${test_file} tester_ip=${tester_ip}" ; then
+         echo "-- Failed to set up test!"
```

Comment:
I'm fine with making this change, but it would actually constitute a behavior change as compared to the master branch's behavior. Does it make sense to lump this behavior change in with this particular PR, or does it make sense to follow up with a separate one?

Reply:
run-tester.yml is "infrastructure" so it should "never" fail, I think.  I agree with the ovn_tester.py part below.